### ISSUE TITLE
Prepare against specific opponent with opening builder

### ISSUE THREAD

**Issue by alexandru-duca** [marcusbuffett/chessmadra-frontend#11]
The opening builder is a great tool to prepare against a random opponent in your elo range. But what if you wanted to prepare for a specific opponent? The probabilities of getting into certain lines against a specific opponent may be completely different compared to the average opponent, e.g. your opponent may be an 1. b3 player. I suggest the following feature:

In the settings, you can choose between "general" and "specific" preparation. In the general case, you can choose the elo range as usual. In the specific case, you can enter the lichess username of the opponent you want to prepare against.

**Comment by marcusbuffett** [comment]
Absolutely, I’ve thought along similar lines. Definitely plan to add this feature
